vkcs/kubernetes/shared.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/util/errutil"
 )
 
+func clusterMutexKey(clusterUUID string) string {
```

Comment:
Looks like you use it only in one file for node group resource, so it would be better to place it there.

Reply:
done